app/tailwind

```diff
@@ -0,0 +1,2201 @@
+/*
```

Comment:
did you touched this file?

Reply:
I was trying to modify some declarations to see if it would solve the "flickering CSS" issue. (The suggested by the community) 
However, i didn't save any of those changes. The file holds its original content.